src/balances/entities/__tests__/fiat-codes.factory.ts

```diff
@@ -0,0 +1,3 @@
+export default function (): string[] {
+  return ['AED', 'AFN', 'EUR', 'ALL', 'USD'];
+}
```

Comment:
Since this is only used in one place I'd just use this payload in the test itself. It also makes it easier to read because you can see what the function returns and what is expected in the same function.

These factory functions would probably live closer to the data involved (for this one probably under `src/services/exchange/entities/__tests__/` since then the factory should be useful for other modules to use)